packages/executor/src/services/EntryPointService/versions/0.0.7.ts

```diff
@@ -70,11 +78,22 @@ export class EntryPointV7Service implements IEntryPointService {
           this.networkConfig.estimationGasLimit
         )
       : undefined;
+    
+    const estimateCallGasArgs: CallGasEstimationProxy.EstimateCallGasArgsStruct = {
+      userOp: packUserOp(userOp),
+      isContinuation: true,
+      maxGas: "100000",
+      minGas: "21000",
+      rounding: "2"
```

Comment:
pls provide justification for these values too

Reply:
i was wondering if we can use 21000 for minGas (as this is the lowest gas for any tx), blockGasLimit for maxGas and increasing the rounding by 1k maybe, which will reduce the number of iterations, but this would also mean that estimations provided will only be precise till 1k.

Also, i have not accounted for these 2 errors:
- `error EstimateCallGasContinuation(uint256 minGas, uint256 maxGas, uint256 numRounds);` (we will have to make another eth_call with max and min values with which it was reverted).
- `error EstimateCallGasRevertAtMax(bytes revertData);` (this will be mitigated if we use blockGasLimit)